modules/oe_theme_helper/tests/src/Functional/Plugin/field_group/FieldInPageNavigationFormatterTest.php

```diff
@@ -0,0 +1,284 @@
+    $this->container->get('config.factory')
+      ->getEditable('system.theme')
+      ->set('default', 'oe_theme')
+      ->save();
+
+    // Rebuild the ui_pattern definitions to collect the ones provided by
+    // oe_theme itself.
+    $this->container->get('plugin.manager.ui_patterns')->clearCachedDefinitions();
+
+    // Create content type.
+    $this->drupalCreateContentType([
+      'name' => 'Test',
+      'type' => 'test',
+    ]);
+
+    /** @var \Drupal\Core\Entity\Display\EntityViewDisplayInterface $display */
+    $display = \Drupal::entityTypeManager()
+      ->getStorage('entity_view_display')
+      ->load('node.test.default');
+
+    // Create test fields.
+    $this->fields = [
+      'field_test_1' => [
+        'content' => 'Field 1 content',
+        'label' => 'Field 1 label',
+        'visibility' => 'above',
+        'type' => 'string',
+        'weight' => 11,
+      ],
+      'field_test_2' => [
+        'content' => 'Field 2 content',
+        'label' => 'Field 2 label',
+        'visibility' => 'inline',
+        'type' => 'string',
+        'weight' => 21,
+      ],
+      'field_test_3' => [
+        'content' => '',
+        'label' => 'Field 3 label',
+        'visibility' => 'inline',
+        'type' => 'string',
+        'weight' => 31,
+      ],
+      'field_test_4' => [
+        'content' => 'Field 4 content',
+        'label' => 'Field 4 label',
+        'visibility' => 'above',
+        'type' => 'string',
+        'weight' => 41,
+      ],
+      'field_test_5' => [
+        'content' => 'Field 5 content',
+        'label' => 'Field 5 label',
+        'visibility' => 'above',
+        'type' => 'string',
+        'weight' => 61,
+      ],
+      'field_test_6' => [
+        'content' => 'Field 6 content',
+        'label' => 'Field 6 label',
+        'visibility' => 'hidden',
+        'type' => 'string',
+        'weight' => 62,
+      ],
+    ];
+
+    foreach ($this->fields as $field_name => $field) {
+      $field_storage = FieldStorageConfig::create([
+        'field_name' => $field_name,
+        'entity_type' => 'node',
+        'cardinality' => 1,
+        'type' => 'text',
+      ]);
+      $field_storage->save();
+
+      $instance = FieldConfig::create([
+        'field_storage' => $field_storage,
+        'bundle' => 'test',
+        'entity_type' => 'node',
+        'settings' => [],
+        'label' => $field['label'],
+        'required' => FALSE,
+      ]);
+      $instance->save();
+
+      // Set the field visible on the display object.
+      $display->setComponent($field_name, [
+        'label' => $field['visibility'],
+        'type' => 'text_default',
+        'weight' => $field['weight'],
+      ]);
+    }
+    // Save display + create node.
+    $display->save();
+  }
+
+  /**
+   * Tests the in-page navigation field group formatter.
+   */
+  public function testFieldInPageNavigationFormmater(): void {
```

Comment:
```suggestion
  public function testFormmater(): void {
```
No need to repeat the whole class name in the method name.